db/migrate/20150812080320_add_picture_assignments_and_picture_styles.rb

```diff
@@ -0,0 +1,34 @@
+class AddPictureAssignmentsAndPictureStyles < ActiveRecord::Migration
+  def change
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for change is too high. [21/15]